app/uk/gov/hmrc/vatapi/config/AppContext.scala

```diff
@@ -19,50 +19,40 @@ package uk.gov.hmrc.vatapi.config
-  lazy val serviceLocatorUrl: String = baseUrl("service-locator")
+  lazy val appName: String = config.getString("appName")
+  lazy val appUrl: String = config.getString("appUrl")
+  lazy val apiGatewayContext: String = config.getString("api.gateway.context")
+  lazy val apiGatewayRegistrationContext: String = apiGatewayContext
+  lazy val apiGatewayLinkContext: String = apiGatewayContext
+  lazy val registrationEnabled: Boolean = current.configuration.getBoolean(s"microservice.services.service-locator.enabled").getOrElse(true)
+  lazy val serviceLocatorUrl: String = config.baseUrl("service-locator")
 
   //DES Config
-  lazy val desEnv: String = config.getString(s"$env.microservice.services.des.env").getOrElse(throw new RuntimeException("desEnv is not configured"))
-  lazy val desToken: String = config.getString(s"$env.microservice.services.des.token").getOrElse(throw new RuntimeException("desEnv is not configured"))
-  lazy val desUrl: String = baseUrl("des")
+  lazy val desEnv: String = config.getString(s"microservice.services.des.env")
+  lazy val desToken: String = config.getString(s"microservice.services.des.token")
+  lazy val desUrl: String = config.baseUrl("des")
   //NRS Config
-  lazy val nrsServiceUrl: String = baseUrl("non-repudiation")
-  lazy val xApiKey: String = config.getString(s"$env.access-keys.xApiKey").getOrElse(throw new RuntimeException("X-API-Key is not configured"))
-  lazy val nrsMaxTimeoutMillis: Int = config.getInt(s"$env.microservice.services.non-repudiation.maxTimeout").getOrElse(throw new RuntimeException("NRS maximum timeout is not configured"))
+  lazy val nrsServiceUrl: String = config.baseUrl("non-repudiation")
+  lazy val xApiKey: String = config.getString(s"access-keys.xApiKey")
+  lazy val nrsMaxTimeoutMillis: Int = config.getInt(s"microservice.services.non-repudiation.maxTimeout")
 
-  lazy val featureSwitch: Option[Configuration] = config.getConfig(s"$env.feature-switch")
+  lazy val featureSwitch: String = config.getString(s"feature-switch")
 
-  lazy val auditEnabled: Boolean = config.getBoolean(s"auditing.enabled").getOrElse(true)
-  lazy val authEnabled: Boolean = config.getBoolean(s"$env.microservice.services.auth.enabled").getOrElse(true)
-  lazy val vatAuthEnrolments: VATAuthEnrolments = VATAuthEnrolments(config.getString(s"$env.enrolments.key").getOrElse(throw new RuntimeException("enrolments.key is not configured")),
-    config.getString(s"$env.enrolments.identifier").getOrElse(throw new RuntimeException("identifier is not configured")),
-    config.getString(s"$env.enrolments.auth-rule"))
-  lazy val vatHybridFeatureEnabled = config.getBoolean(s"$env.feature-switch.des.hybrid").getOrElse(false)
+  lazy val vatAuthEnrolments: VATAuthEnrolments = VATAuthEnrolments(config.getString(s"enrolments.key"),
+    config.getString(s"enrolments.identifier"),
+    Some(config.getString(s"enrolments.authRule")))
```

Comment:
This is optional in VATAuthEnrolments (with code to default the value). Why not use configuration.getOptional[String] instead? (Risks throwing exception if this is not present in config.)